meta-ostro-xt/recipes-opencl/ocl-icd/ocl-icd/configurable-doc-generation.patch

```diff
@@ -0,0 +1,25 @@
+diff --git a/configure.ac b/configure.ac
```

Comment:
@mythi, Currently there are two patches in kernel patch series on XT:
1. librealsense_formats_linux-yocto_4.4.patch
2. 0001-ovl-setxattr-don-t-deadlock-when-called-from-ima_fix.patch

Both fail to apply if we not provide committer details either via environment or via global git config.
This is not visible on non-docker builds as, usually host .gitconfig set properly, But not the case with docker container.